stl/src/syserror_import_lib.cpp

```diff
@@ -10,7 +10,7 @@
 
 #include <Windows.h>
 
-namespace {
+/* namespace {

```

Comment:
Don't comment out the code, just remove it if the code is not needed.

Reply:
Removed dosen't Neaded Code